src/other/Opportunity.css

```diff
@@ -0,0 +1,104 @@
+.graph_Opportunity {
+    transform: translate(20px,20px)scale(1) !important;
+}
+
+#svgcontainer {
+    width:950px;
+    text-align:center;
+}
+
+ .group {
+    stroke-width: 1.0;
+    stroke:#555;
+    fill:#FEFEFE;
+}
+
+.graph_Opportunity .group_headings {
+    font-family: "Helvetica","Arial",sans-serif;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 20px;
+}
+
+.graph_Opportunity .node_prev_rect {
+    fill:white;
+    stroke-width:2;
+    stroke:#000;
+    stroke-dasharray:3,3;
+      opacity:0.6;
+}}
+
+.graph_Opportunity .node_rev_change {
+    fill:white;
+}
+.graph_Opportunity .node_rev_change_text {
+  font-size:18px;
+  font-family:Arial;
+  color: black;
+  font-weight: bold;
+
+}
+
+.graph_Opportunity .node_date_change {
+    fill:white;
+}
+
+.graph_Opportunity .node_date_change_text {
+  font-size:18px;
+  font-family:Arial;
+  color: black;
+  font-weight: bold;
+}
+
+.graph_Opportunity .node_prev_text {
+    font-size:9px;
+    font-family:Arial;
+    opacity:0.6;
+}
+
+.graph_Opportunity .node_cur_rect {
+    stroke-width:2;
+    stroke:#000;
+}
+
+.graph_Opportunity .node_cur_text {
+    font-size:9px;
+    font-family:Arial;
+    text-decoration: underline;
+   
+}
+
+.graph_Opportunity .arrowhead {
+    refX:6;
+}
+
+div.tooltip {
```

Comment:
You should add a ```.graph_Opportunity-tooltip``` class to your tooltip and then use that (this style will potentially clash with users tooltips)